### ISSUE TITLE
[BUG] applymap error from the doc string example

### ISSUE THREAD

**Issue by daxiongshu** [rapidsai/cudf#6667]
**Describe the bug**
There is an unexpected error when running an example from `applymap`'s doc string. @hummingtree 

Code:
```
import cudf
cudf.__version__ # 0.16.0
dt = cudf.DataFrame({'a':[1,1,1]})
dt['a'].applymap(lambda x: x in [1,2,3])
```

Bug:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-34-ad1a4432195b> in <module>
      3 
      4 dt = cudf.DataFrame({'a':[1,1,1]})
----> 5 dt['a'].applymap(lambda x: x in [1,2,3])
~/anaconda3/envs/rapids-0.16/lib/python3.7/site-packages/cudf/core/series.py in applymap(self, udf, out_dtype)
   2565 
   2566         if callable(udf):
-> 2567             res_col = self._unaryop(udf)
   2568         else:
   2569             res_col = self._column.applymap(udf, out_dtype=out_dtype)
~/anaconda3/envs/rapids-0.16/lib/python3.7/site-packages/cudf/core/frame.py in _unaryop(self, op)
   2124         data_columns = (col.unary_operator(op) for col in self._columns)
   2125         data = zip(self._column_names, data_columns)
-> 2126         return self.__class__._from_table(Frame(data, self._index))
   2127 
   2128     def isnull(self):
cudf/_lib/table.pyx in cudf._lib.table.Table.__init__()
~/anaconda3/envs/rapids-0.16/lib/python3.7/site-packages/cudf/core/column_accessor.py in __init__(self, data, multiindex, level_names)
     38             self._level_names = level_names
     39 
---> 40         self._data = OrderedColumnDict(data)
     41         self.multiindex = multiindex
     42         self._level_names = level_names
~/anaconda3/envs/rapids-0.16/lib/python3.7/site-packages/cudf/core/frame.py in <genexpr>(.0)
   2122 
   2123     def _unaryop(self, op):
-> 2124         data_columns = (col.unary_operator(op) for col in self._columns)
   2125         data = zip(self._column_names, data_columns)
   2126         return self.__class__._from_table(Frame(data, self._index))
~/anaconda3/envs/rapids-0.16/lib/python3.7/site-packages/cudf/core/column/numerical.py in unary_operator(self, unaryop)
     67 
     68     def unary_operator(self, unaryop):
---> 69         return _numeric_column_unaryop(self, op=unaryop)
     70 
     71     def binary_operator(self, binop, rhs, reflect=False):
~/anaconda3/envs/rapids-0.16/lib/python3.7/site-packages/cudf/core/column/numerical.py in _numeric_column_unaryop(operand, op)
    573 def _numeric_column_unaryop(operand, op):
    574     if callable(op):
--> 575         return libcudf.transform.transform(operand, op)
    576 
    577     op = libcudf.unary.UnaryOp[op.upper()]
cudf/_lib/transform.pyx in cudf._lib.transform.transform()
RuntimeError: CUDA_ERROR_INVALID_PTX
```

**Environment overview (please complete the following information)**
 - Environment location: Bare-metal 
- Method of cuDF install: conda

**Comment by hummingtree** [comment]
This is caused by a bug in the PTX parser and should be quick fix.

**Comment by kkraus14** [comment]
Fixed by #6670
